src/parser/user-extractor-transformer.ts

```diff
@@ -0,0 +1,68 @@
+import configuration from "../configuration/config-reader";
+import { GetActivity } from "../get-activity";
+import { GitHubIssue, GitHubIssueComment } from "../github-types";
+import { Result, Transformer } from "./processor";
+
+/**
+ * Creates entries for each bounty hunter with its associated comments.
+ */
+export class UserExtractorTransformer implements Transformer {
+  private readonly _configuration = configuration["user-extractor"];
+
+  get enabled(): boolean {
+    return true;
+  }
+
+  /**
+   * Checks if the comment is made by a human user, and if it not a command.
+   * @param comment
```

Comment:
```suggestion
```

I feel that JSDoc is net negative in a TypeScript environment because its easy to update function signatures and forget to update the comments. Besides, the function signatures clearly define the data types and the parameters?

Reply:
My IDE auto generates it and gives me warnings if it doesn't match the defined function that's why it's there, I can remove it

Reply:
Vscode doesn't do any of that by default so it seems to be a liability.